circle.yml

```diff
@@ -13,6 +13,7 @@ dependencies:
     - sudo apt-get install python-numpy python-scipy python-dev python-matplotlib
     - sudo apt-get install python-nose python-coverage
     - sudo apt-get install python-sphinx
+    - pip install --find-links http://wheels.astropy.org/ --find-links http://wheels2.astropy.org/ --use-wheel Cython
```

Comment:
I don't understand. It's right here https://pypi.python.org/pypi/Cython/ so why not just pip install the latest version?

Also out of the two links, one has the Cython version 0.19.1 and the other link raises a 404 error.